notebook/notebookapp.py

```diff
@@ -1490,7 +1490,10 @@ def init_webapp(self):
                 self.http_server.listen(port, self.ip)
             except socket.error as e:
                 if e.errno == errno.EADDRINUSE:
-                    self.log.info(_('The port %i is already in use, trying another port.') % port)
+                    if self.port_retries:
+                        self.log.info(_('The port %i is already in use, trying another port.') % port)
+                    else:
+                        self.log.info(_('The port %i is already in use') % port)
```

Comment:
nit: Let's add the period.  Since the permission warning doesn't have one, yet the other messages do, please add a period to that warning message as well.  Thanks.

Reply:
Sure. I've updated the commit, thank you.